src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss

```diff
@@ -7,8 +7,10 @@ $itemsContainerBorderColor: $gallery;
 
 $itemsContainerBorder: 1px $itemsContainerBorderColor solid;
 
+$toolbarHeight: 60px;
```

Comment:
Isn't really consistent with the `variables.scss` file above... But the above variable isn't used in its own bundle.

Reply:
ack